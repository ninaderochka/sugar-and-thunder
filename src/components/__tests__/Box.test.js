import React from 'react';
import renderer from 'react-test-renderer';
import Box from '../Box';

test('Box component renders correctly', () => {
  const component = renderer.create(
    <Box>
      <p>Example child content</p>
    </Box>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


