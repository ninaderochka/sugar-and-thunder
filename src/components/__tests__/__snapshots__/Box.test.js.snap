// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component renders correctly 1`] = `
<div
  className="bg-white border-border-grey shadow-shadow-grey drop-shadow-lg font-poppins font-normal box-border p-4 border-2 rounded-[20px] flex content-center m-auto aspect-square w-9/12 md:w-box-width md:h-box-height"
>
  <p>
    Example child content
  </p>
</div>
`;
